src/main/java/org/opentripplanner/api/common/Message.java

```diff
@@ -26,6 +26,7 @@ public enum Message {
   GEOCODE_FROM_NOT_FOUND(440),
   GEOCODE_TO_NOT_FOUND(450),
   GEOCODE_FROM_TO_NOT_FOUND(460),
+  GEOCODE_INTERMEDIATE_NOT_FOUND(465),
```

Comment:
I know that the rest of the errors also have the same name but this error has nothing to do with geocoding. Should we perhaps not make it worse by giving a better name?

Reply:
would it make sense to rename them all in another PR with a new prefix TRIP or TRIP_SEARCH?